exes/mqtt-broadcaster/inc/config/spark_plug_b_config.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <tfc/confman.hpp>
+
+namespace tfc::mqtt {
+
+// File under /etc/tfc/mqtt-broadcaster/def/mqtt_broadcaster.json which specifies the connection values for the MQTT broker.
+struct spark_plug_b_config {
+  std::string node_id{};
+  std::string group_id{};
+
+  struct glaze {
+    // clang-format off
+    static constexpr auto value{ glz::object(
+         "node_id", &spark_plug_b_config::node_id, "Spark Plug B Node ID, used to identify which node is sending information",
+         "group_id", &spark_plug_b_config::group_id, "Spark Plug B Group ID, used to identify which group the node belongs to"
```

Comment:
The string must be a valid utf-8 string with the exception of +, /, and #. 

https://sparkplug.eclipse.org/specification/version/3.0/documents/sparkplug-specification-3.0.0.pdf, page 26. 

This applies to both node id and group id. I can look into `pattern` to see if it can be used.